cumulusci/core/config/project_config.py

```diff
@@ -835,3 +835,7 @@ def construct_subproject_config(self, **kwargs):
     def relpath(self, path):
         """Convert path to be relative to the project repo root."""
         return os.path.relpath(os.path.join(self.repo_root, path))
+
+    @property
+    def project_cache_dir(self):
+        return Path(self.repo_root, ".cci")
```

Comment:
There are a couple of existing bits of code that have ".cci" hardcoded -- let's go ahead and update them to use this.

Reply:
I'll submit a PR for that but it balloons.

"change this to a call to the project_config.project_cache_dir"

Now the tests need project_config.repo_root to be set, or some of them fail.

The "right" way to set that is with a fixture I created a few weeks ago.

But fixtures can only be used in UnitTest-style tests.

And UnitTest-style tests have a different assertion syntax.

Reply:
You mean pytest fixtures can only be used in test classes that don't extend unittest.TestCase.